merge.go

```diff
@@ -162,6 +162,28 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
 	return mergeCells, err
 }
 
+// GetMergedCellRefs provides a function to get all merged cells' range
```

Comment:
Add an optional parameter `withoutValues` for the existing function `GetMergeCells` instead of introduction new function will be better approach.